test/e2e/csv_e2e_test.go

```diff
@@ -401,6 +401,10 @@ var _ = Describe("ClusterServiceVersion", func() {
 	})
 
 	Context("AllNamespaces OperatorGroup", func() {
+		BeforeEach(func() {
+			nsName := genName("csv-e2e-")
+			generatedNamespace = SetupGeneratedTestNamespace(nsName)
```

Comment:
Why is this valid? Were the tests reusing one namespace in the past?

Reply:
This is actually a no-op change, so I'm going to revert it with the exception of changing  `ns =` to `generatedNamespace =`